logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/MessageSender.kt

```diff
@@ -246,7 +246,8 @@ internal class MessageSenderImpl internal constructor(
 
     private suspend fun attemptToSendWithProteus(
         message: Message.Sendable,
-        messageTarget: MessageTarget
+        messageTarget: MessageTarget,
+        retryCount: Int = 0
```

Comment:
can you please add a comment about the retry and how it works

Reply:
I think retry attempt is self explanatory